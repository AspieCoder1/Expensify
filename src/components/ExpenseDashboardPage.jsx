import React from 'react';
import Header from './Header.jsx';
import ConnectedExpenseList from './ExpenseList.jsx';
import ExpenseListFilters from './ExpenseListFilters.jsx';

const ExpenseDashboardPage = () => (
	<div>
		<Header />
		<ExpenseListFilters />
		<ConnectedExpenseList />
	</div>
);

export default ExpenseDashboardPage; 