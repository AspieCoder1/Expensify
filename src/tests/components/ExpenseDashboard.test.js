import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage.jsx';

test('shoulds render ExpenseDashboardPage correctly', () => {
	const wrapper = shallow(<ExpenseDashboardPage />);
	expect(wrapper).toMatchSnapshot();
});
