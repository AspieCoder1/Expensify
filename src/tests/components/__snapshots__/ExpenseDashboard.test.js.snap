// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoulds render ExpenseDashboardPage correctly 1`] = `
<div>
  <Header />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
